src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_vload8.cpp

```diff
@@ -141,7 +143,7 @@ KernelsData EltwiseKernel_vload8::GetKernelsData(const Params& params, const opt
     return {kd};
 }
 
-KernelsPriority EltwiseKernel_vload8::GetKernelsPriority(const Params& /*params*/, const optional_params& /*options*/) const {
-    return FORCE_PRIORITY_8;
+KernelsPriority EltwiseKernel_vload8::GetKernelsPriority(const Params& params, const optional_params& /*options*/) const {
+    return FORCE_PRIORITY_1;
```

Comment:
why did you change priority of this kernel?

Reply:
if the model has no broadcasting, no multiple post-ops and divided by 8 in feature axis. I thought this kernel would have the best perf with the given lws (except b_fs_yx_fsv16 with sub_group_block). I tested this with unit-tests of fs_b_yx_fsv32. 
Is there a point to check? or Is there anything I missed about this change?